Gadgets/MatchingGadgets.h

```diff
@@ -670,31 +434,37 @@ class OrderMatchingGadget : public GadgetT
         checkValidA.generate_r1cs_constraints();
         checkValidB.generate_r1cs_constraints();
         valid.generate_r1cs_constraints();
+        requireValid.generate_r1cs_constraints();
     }
 
     const VariableT& getFillA_S() const
```

Comment:
Deprecated, we can remove it totally.

Reply:
Good point. `getFillA_S()` etc... are not used any more so it's better it's removed.